docbook/reference/en/en-US/modules/RESTEasy_Client_Framework.xml

```diff
@@ -1059,8 +1059,25 @@ ResteasyClient client = ((ResteasyClientBuilder)ClientBuilder.newBuilder())
         </programlisting>
         <para>
             When coupled with the MonoRxInvoker, this has several benefits.
-            Most notably, a Reactor-based timeout will eagerly stop in-flight
-            requests.
+            It supports things like this:

```

Comment:
I like this change.

As an aside, I'm not sure how you got this into my PR.  I wasn't aware anyone could push to my repo:P  But.. you are welcome to! :)

Reply:
> I'm not sure how you got this into my PR

I guess I have some extra administrative powers because I can edit a file and commit it right on the spot. It's very convenient. I'm not sure what would happen if I tried to push to your repo. Sometimes I've been able to do that, and sometimes not.

Old Lodge Skins: Am I still in this world?

Jack Crabb: Yes, Grandfather.

Old Lodge Skins: [groans] I was afraid of that. Well, sometimes the magic works. Sometimes, it doesn't.